core/server/data/migrations/versions/4.0/15-add-frontmatter-column-to-meta.js

```diff
@@ -0,0 +1,7 @@
+const {createAddColumnMigration} = require('../../utils');
+
+module.exports = createAddColumnMigration('posts_meta', 'frontmatter', {
+    type: 'text',
+    maxlength: 65535,
+    nullable: false
```

Comment:
`nullable: false` vs `nullable: true` in schema definition 🙂

Reply:
good catch 🙊

Reply:
fixed ✔️